### PR TITLE
show release alias on rules page

### PR DESCRIPTION
**Pull request by nazarenegena**
This PR attempts to fix the issue #1138.
I have added the alias for the releases on the rules page. When you hover on the specific rule you can see it's alias.
however, since most of the releases' alias is 'null' I have left the '#' in its place. 
Is this the expected solution?

### CODE REVIEW

**Review by bhearsum — CHANGES_REQUESTED** [review]
Unfortunately, this does not address the issue. What we'd like is for a change on the Releases page to show rule alias' instead of rule ids. For example, in this screenshot:
image

The `56` should be `firefox-release-localtest`, `145` should be `firefox-release`, and `585` should be `firefox-release-cdntest`.

This will require changes to https://github.com/mozilla-releng/balrog/blob/main/ui/src/components/ReleaseCard/index.jsx and possibly some of the data passed to it.

**Comment by nazarenegena** [comment]
Cool, thanks for the update. I will work on it.

**Comment by nazarenegena** [comment]
Updated the releases page to now show rule alias instead of rule ids

File: ui/src/components/ReleaseCard/index.jsx [hunk 1]
```diff
@@ -254,7 +254,9 @@ function ReleaseCard(props) {
                               clickable
                               size="small"
                               icon={<LinkIcon />}
-                              label={ruleId}
+                              label={`${ruleInfo.product} ${
+                                ruleInfo.channel ? `- ${ruleInfo.channel}` : ''
```
Comment (bhearsum) [inline ui/src/components/ReleaseCard/index.jsx:258]: This is not what we want either. The alias is a property of the `rule`. I can see why you're struggling though - the alias is not returned by the `/releases` endpoint, so the data you need is not even present.

I suggest you fetch the rules (similar to how we do on the rules page: https://github.com/mozilla-releng/balrog/blob/ca55f9aa6aa15647370a841d8827e1e724ab8813/ui/src/views/Rules/ListRules/index.jsx#L164), and use that to find the alias for each ruleId.

**Review by bhearsum — CHANGES_REQUESTED** [review]

**Comment by nazarenegena** [comment]
Cool, Noted. Will work on this.

**Comment by nazarenegena** [comment]
I have updated the alias for the releases on the rules page
As for the rules that don't have an alias I have used their rule id

**Review by bhearsum — CHANGES_REQUESTED** [review]
The changes look good - can you please fix the lint issues before we merge, though?

**Comment by nazarenegena** [comment]
Working on the fixes.

**Comment by nazarenegena** [comment]
I have fixed the lint issues

**Review by bhearsum — APPROVED** [review]
Thank you!

**Comment by bhearsum** [comment]
@nazarenegena - Unfortunately, this now has conflicts with the latest `main`. Can you update your branch? Feel free to ping me on Matrix if you need help with that.

**Comment by nazarenegena** [comment]
Cool, let me update my branch.

**Comment by nazarenegena** [comment]
conflicts resolved.

**Review by bhearsum — APPROVED** [review]
This looks great now - thank you!

**Comment by nazarenegena** [comment]
you're welcome.

**Pull request merged**